pom.xml

```diff
@@ -792,7 +789,7 @@
         <plugin>
           <groupId>org.owasp</groupId>
           <artifactId>dependency-check-maven</artifactId>
-          <version>8.3.1</version>
+          <version>12.1.3</version>
```

Comment:
I experienced error in 'mvn dependency-check:check' in old version.

Reply:
This is being addressed in #2297